tests/admin_scripts/tests.py

```diff
@@ -1901,10 +1902,16 @@ def test_help_default_options_with_custom_arguments(self):
         ]
         for option in expected_options:
             self.assertOutput(out, f"[{option}]")
-        self.assertOutput(out, "--option_a OPTION_A, -a OPTION_A")
-        self.assertOutput(out, "--option_b OPTION_B, -b OPTION_B")
-        self.assertOutput(out, "--option_c OPTION_C, -c OPTION_C")
-        self.assertOutput(out, "-v {0,1,2,3}, --verbosity {0,1,2,3}")
+        if PY313:
+            self.assertOutput(out, "--option_a, -a OPTION_A")
+            self.assertOutput(out, "--option_b, -b OPTION_B")
+            self.assertOutput(out, "--option_c, -c OPTION_C")
+            self.assertOutput(out, "-v, --verbosity {0,1,2,3}")
+        else:
+            self.assertOutput(out, "--option_a OPTION_A, -a OPTION_A")
+            self.assertOutput(out, "--option_b OPTION_B, -b OPTION_B")
+            self.assertOutput(out, "--option_c OPTION_C, -c OPTION_C")
+            self.assertOutput(out, "-v {0,1,2,3}, --verbosity {0,1,2,3}")
```

Comment:
Since the last assert did not change, we could move it outside the if-else and remove the duplicated one?
```suggestion
    self.assertOutput(out, "-v {0,1,2,3}, --verbosity {0,1,2,3}")
```

Reply:
It's different:
```
"-v, --verbosity {0,1,2,3}"
```
vs.
```
"-v {0,1,2,3}, --verbosity {0,1,2,3}"
```

Reply:
It did change tho 🤔 :)

Reply:
You are right, I misread.